Source/NSDataAsset.m

```diff
@@ -0,0 +1,85 @@
+/* Implementation of class NSDataAsset
+   Copyright (C) 2019 Free Software Foundation, Inc.
+   
+   By: Gregory John Casamento
+   Date: Fri Jan 17 10:25:34 EST 2020
+
+   This file is part of the GNUstep Library.
+   
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2 of the License, or (at your option) any later version.
+   
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+   
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; if not, write to the Free
+   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110 USA.
+*/
+
+#import <AppKit/NSDataAsset.h>
```

Comment:
I prefer to have the Foundation imports first but this is just a matter of taste.

Reply:
Done.